picard/util/__init__.py

```diff
@@ -442,15 +442,14 @@ def union_sorted_lists(list1, list2):
     return union
 
 
-def htmlescape(string):
-    return cgi.escape(string)
-
-
-def json_load(data):
-    return json.loads(bytes(data).decode())
-
-
 def convert_to_string(obj):
+    """
+    Appropriately converts the input `obj` to a string.
```

Comment:
Looking at the documentation for google style docstrings, you're missing a blank line between each section here and in `json_load`.